docs/source/howto/installation.rst

```diff
@@ -547,17 +547,26 @@ See the :doc:`../reference/_changelog` for a list of breaking changes.
 
 .. _how-to:installation:backup:
 
-Backing up your installation
+Backing up your data
 ============================
 
-A full backup of an AiiDA instance and AiiDA managed data requires a backup of:
+The most convenient way to back up the data of a single AiiDA profile is to use
```

Comment:
I think we already need to separate this in two parts:
* generic info
* storage backend specific

The generic stuff can mention the CLI and API commands to backup a profile, but should mention that the specifics, and whether it is even implemented, depends on the backend.

For storage backend specific stuff we could use tabs for example, like in the installation section, one tab for each storage backend in `aiida-core`. We can keep the `core.psql_dos` as the first one since that will be the most common case.

Reply:
Note that the "Restore" section already uses tabs as well, so we just need to apply that here as well.

Reply:
thanks! updated accordingly